### ISSUE TITLE
[Bug] After placing an 8th End Stone Stalactite going down to connect it to another 8 End Stone Stalactites going up, the game crashes.

### ISSUE THREAD

**Issue by GamingWithDudeAlt** [quiqueck/BetterEnd#260]
### What happened?

The title is what happened, but this exact same thing also happens with the Cave Moss Endstone Stalactite. What I expected to happen was them connecting to each other with no issues like with every other stalactite in the screenshot attached. I've attached the latest.logs below aswell because I couldn't post with them in the "Relevant log output" section.
![Screenshot (6480)](https://github.com/quiqueck/BetterEnd/assets/135929375/8b7b8503-5f4e-485c-ae71-7ea275e738e3)
[latest.log](https://github.com/quiqueck/BetterEnd/files/11970731/latest.log)
[latest (1).log](https://github.com/quiqueck/BetterEnd/files/11970732/latest.1.log)

### BetterEnd

4.0.7

### BCLib

3.0.10

### Fabric API

0.85.0

### Fabric Loader

0.14.21

### Minecraft

1.20.1

### Other Mods

```shell
[12:11:27] [main/INFO]: Loading 226 mods:
	- alternate-current 1.7.0
	- ambientsounds 5.2.20
	- appleskin 2.5.0+mc1.20
	- architectury 9.0.8
	- betteradvancements 0.3.2.160
	- betterarcheology 1.0.2
	- betterfpsdist 1.20.1-2.7
	- betternether 9.0.7
	- biomemusic 1.20.1-1.6
	- cardinal-components 5.2.1
	   |-- cardinal-components-base 5.2.1
	   |-- cardinal-components-block 5.2.1
	   |-- cardinal-components-chunk 5.2.1
	   |-- cardinal-components-entity 5.2.1
	   |-- cardinal-components-item 5.2.1
	   |-- cardinal-components-level 5.2.1
	   |-- cardinal-components-scoreboard 5.2.1
	   \-- cardinal-components-world 5.2.1
	- cavedust 1.4.1
	   \-- kirin 1.15.0
	- charmofundying 6.4.2+1.20.1
	   \-- spectrelib 0.13.13+1.20.1
	- chunksending 1.20.1-2.5
	- citresewn 1.1.3+1.20
	   \-- citresewn-defaults 1.1.3+1.20
	- cloth-config 11.0.99
	   \-- cloth-basic-math 0.6.1
	- clumps 12.0.0.3
	- comforts 6.3.3+1.20.1
	   |-- cardinal-components-base 5.2.1
	   \-- cardinal-components-entity 5.2.1
	- completionistsindex 8.0.0
	- continuity 3.0.0-beta.2+1.20
	- controlling 12.0.1
	- creativecore 2.10.24
	   \-- net_minecraftforge_eventbus 6.0.3
	- dark-loading-screen 1.6.14
	- dawn 5.0.0
	   |-- terraform-shapes-api-v1 7.0.1
	   \-- terraform-wood-api-v1 7.0.1
	- debugify 1.20.1+1.1
	   \-- com_github_llamalad7_mixinextras 0.2.0-beta.8
	- dynamicfps 2.4.0
	   \-- com_moandjiezana_toml_toml4j 0.7.2
	- eatinganimationid 1.9.4+1.20
	- enhancedblockentities 0.9+1.20
	   |-- advanced_runtime_resource_pack 0.6.7
	   \-- spruceui 5.0.0+1.20
	- entity_model_features 0.2.11
	- entity_texture_features 4.4.4
	   \-- org_apache_httpcomponents_httpmime 4.5.10
	- entityculling 1.6.2-mc1.20
	- fabric-api 0.85.0+1.20.1
	   |-- fabric-api-base 0.4.29+b04edc7a77
	   |-- fabric-api-lookup-api-v1 1.6.34+4d8536c977
	   |-- fabric-biome-api-v1 13.0.10+b3afc78b77
	   |-- fabric-block-api-v1 1.0.9+e022e5d177
	   |-- fabric-blockrenderlayer-v1 1.1.39+b3afc78b77
	   |-- fabric-client-tags-api-v1 1.1.0+97bb207577
	   |-- fabric-command-api-v1 1.2.32+f71b366f77
	   |-- fabric-command-api-v2 2.2.11+b3afc78b77
	   |-- fabric-commands-v0 0.2.49+df3654b377
	   |-- fabric-containers-v0 0.1.61+df3654b377
	   |-- fabric-content-registries-v0 4.0.8+b3afc78b77
	   |-- fabric-convention-tags-v1 1.5.3+b3afc78b77
	   |-- fabric-crash-report-info-v1 0.2.18+aeb40ebe77
	   |-- fabric-data-generation-api-v1 12.1.12+b3afc78b77
	   |-- fabric-dimensions-v1 2.1.51+b3afc78b77
	   |-- fabric-entity-events-v1 1.5.21+b3afc78b77
	   |-- fabric-events-interaction-v0 0.6.0+b3afc78b77
	   |-- fabric-events-lifecycle-v0 0.2.61+df3654b377
	   |-- fabric-game-rule-api-v1 1.0.38+b04edc7a77
	   |-- fabric-item-api-v1 2.1.26+b3afc78b77
	   |-- fabric-item-group-api-v1 4.0.8+40e50c4677
	   |-- fabric-key-binding-api-v1 1.0.36+fb8d95da77
	   |-- fabric-keybindings-v0 0.2.34+df3654b377
	   |-- fabric-lifecycle-events-v1 2.2.20+b3afc78b77
	   |-- fabric-loot-api-v2 1.1.38+b3afc78b77
	   |-- fabric-loot-tables-v1 1.1.42+9e7660c677
	   |-- fabric-message-api-v1 5.1.6+b3afc78b77
	   |-- fabric-mining-level-api-v1 2.1.48+b3afc78b77
	   |-- fabric-models-v0 0.3.35+b3afc78b77
	   |-- fabric-networking-api-v1 1.3.8+b3afc78b77
	   |-- fabric-networking-v0 0.3.48+df3654b377
	   |-- fabric-object-builder-api-v1 11.1.0+6beca84877
	   |-- fabric-particles-v1 1.1.0+201a23a077
	   |-- fabric-recipe-api-v1 1.0.18+b3afc78b77
	   |-- fabric-registry-sync-v0 2.2.6+b3afc78b77
	   |-- fabric-renderer-api-v1 3.1.0+c154966e77
	   |-- fabric-renderer-indigo 1.4.0+c154966e77
	   |-- fabric-renderer-registries-v1 3.2.44+df3654b377
	   |-- fabric-rendering-data-attachment-v1 0.3.33+b3afc78b77
	   |-- fabric-rendering-fluids-v1 3.0.26+b3afc78b77
	   |-- fabric-rendering-v0 1.1.47+df3654b377
	   |-- fabric-rendering-v1 3.0.6+b3afc78b77
	   |-- fabric-resource-conditions-api-v1 2.3.5+ea08f9d877
	   |-- fabric-resource-loader-v0 0.11.8+e3d6ed2577
	   |-- fabric-screen-api-v1 2.0.6+b3afc78b77
	   |-- fabric-screen-handler-api-v1 1.3.27+b3afc78b77
	   |-- fabric-sound-api-v1 1.0.12+b3afc78b77
	   |-- fabric-transfer-api-v1 3.2.3+43a3fedd77
	   \-- fabric-transitive-access-wideners-v1 4.2.0+b3afc78b77
	- fabric-language-kotlin 1.9.6+kotlin.1.8.22
	   |-- org_jetbrains_kotlin_kotlin-reflect 1.8.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib 1.8.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.8.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.8.22
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.21.0
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.7.1
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.7.1
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.4.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.5.1
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.5.1
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.5.1
	- fabricloader 0.14.21
	- fallingleaves 1.15.1+1.20.1
	- fallingtree 4.2.0
	- ferritecore 6.0.0
	- forgeconfigapiport 8.0.0
	- fwaystones 3.1.2+mc1.20
	- geckolib 4.2
	   \-- com_eliotlash_mclib_mclib 20
	- geophilic v2.0.0-mc1.20u1.20.1
	- gpumemleakfix 1.20.1-1.6
	- gravestones v1.15
	- hearths v1.0.0-mc1.20u1.20.1
	- immediatelyfast 1.1.15+1.20.1
	   \-- net_lenni0451_reflect 1.1.0
	- indium 1.0.18+mc1.20
	- inventoryhud 3.4.13
	- iris 1.6.4
	   |-- io_github_douira_glsl-transformer 2.0.0-pre13
	   |-- org_anarres_jcpp 1.4.14
	   \-- org_antlr_antlr4-runtime 4.11.1
	- jade 11.1.4
	- jamlib 0.6.0+1.20
	- java 18
	- kiwi 11.0.0
	- krypton 0.2.3
	   \-- com_velocitypowered_velocity-native 3.2.0-SNAPSHOT
	- lambdynlights 2.3.1+1.20.1
	   |-- pride 1.2.0+1.19.4
	   \-- spruceui 5.0.0+1.20
	- lazydfu 0.1.3
	- letmedespawn fabric-1.20-1.1.0
	- litematica 0.15.3
	- lithium 0.11.2
	- malilib 0.16.1
	- mavapi 1.1.1
	- mavm 1.2.4
	- memoryleakfix 1.1.1
	- midnightlib 1.4.1
	- mindfuldarkness 8.0.0
	- minecraft 1.20.1
	- minihud 0.27.0
	- modernfix 5.1.1+mc1.20.1
	- modmenu 7.1.0
	- moreculling 1.20-0.18.1
	   \-- conditional-mixin 0.3.2
	- moremobvariants 1.2.2
	- mousetweaks 2.25
	- naturescompass 1.20.1-2.2.1-fabric
	- neruina 1.1.0
	- nicer-skies 1.2.0
	- overflowingbars 8.0.0
	- owo 0.11.1+1.20
	   \-- blue_endless_jankson 1.2.2
	- paperdoll 8.0.0
	- pickupnotifier 8.0.0
	- polymorph 0.49.0+1.20.1
	   |-- cardinal-components-base 5.2.1
	   |-- cardinal-components-block 5.2.1
	   |-- cardinal-components-entity 5.2.1
	   |-- cardinal-components-item 5.2.1
	   \-- spectrelib 0.13.13+1.20.1
	- presencefootsteps 1.9.0
	   \-- kirin 1.15.0
	- puzzleslib 8.0.7
	- reacharound 1.1.2
	- reeses-sodium-options 1.5.1+mc1.20-build.74
	- regions_unexplored 0.4.1+1.20.1
	- repurposed_structures 7.0.0+1.20-fabric
	- rightclickharvest 3.2.2+1.19.x-1.20.1-fabric
	- roughlyenoughitems 12.0.626
	   \-- error_notifier 1.0.9
	- searchables 1.0.1
	- servercore 1.3.7+1.20.1
	   |-- com_electronwill_night-config_core 3.6.6
	   |-- com_electronwill_night-config_toml 3.6.6
	   |-- fabric-permissions-api-v0 0.2-SNAPSHOT
	   \-- placeholder-api 2.1.1+1.20
	- smoothchunk 1.20.1-3.0
	- snowrealmagic 9.0.0
	- sodium 0.4.10+build.27
	- sodium-extra 0.4.20+mc1.20-build.103
	   |-- caffeineconfig 1.1.0+1.17
	   \-- crowdin-translate 1.4+1.19.3
	- soulfired 3.2.0.0
	- sound_physics_remastered 1.20.1-1.1.1
	- starlight 1.1.2+fabric.dbc156f
	- structureessentials 1.20.1-2.9
	- terrablender 3.0.0.165
	- tia 1.20-1.1
	- travelersbackpack 1.20.1-9.1.1
	- traverse 7.0.8
	   |-- biolith 1.0.0-alpha.8
	   |    \-- terraform-surfaces-api-v1 7.0.1
	   |-- terraform-biome-remapper-api-v1 7.0.1
	   |-- terraform-config-api-v1 7.0.1
	   |-- terraform-surfaces-api-v1 7.0.1
	   |-- terraform-tree-api-v1 7.0.1
	   |-- terraform-wood-api-v1 7.0.1
	   |-- traverse-client 7.0.8
	   |-- traverse-common 7.0.8
	   \-- traverse-worldgen 7.0.8
	- trinkets 3.7.0
	- universal_ores 1.5.2
	- variantbarrels 3.0
	- variantchiseledbookshelves 1.0
	- waterdripsound 1.19-0.3.2
	- xaeroarrowfix 1.3+1.20
	- xaerominimap 23.5.0
	- xaeroworldmap 1.30.6
	- yet_another_config_lib_v3 3.0.3+1.20
	   |-- com_twelvemonkeys_common_common-image 3.9.4
	   |-- com_twelvemonkeys_common_common-io 3.9.4
	   |-- com_twelvemonkeys_common_common-lang 3.9.4
	   |-- com_twelvemonkeys_imageio_imageio-core 3.9.4
	   |-- com_twelvemonkeys_imageio_imageio-metadata 3.9.4
	   \-- com_twelvemonkeys_imageio_imageio-webp 3.9.4
	- zoomify 2.10.0
	   |-- com_akuleshov7_ktoml-core-jvm 0.4.1
	   |-- dev_isxander_settxi_settxi-core 2.10.6
	   \-- dev_isxander_settxi_settxi-kotlinx-serialization 2.10.6
```

**Comment by quiqueck** [comment]
Thanks, that was a great find.

**Comment by quiqueck** [comment]
This should be fixed with BetterEnd 4.0.8. Please re-open the issue if the problem was not solved for you.
